src/Osma.Mobile.App/App.xaml.cs

```diff
@@ -62,7 +65,9 @@ private async Task Initialize()
 
         protected override void OnStart()
         {
-            // Handle when your app starts
+            AppCenter.Start("ios=aaf709cd-8861-4978-87a3-51885163f6a2" +
```

Comment:
App Secret, should this be brought in from environment variables?

Reply:
Yeap just added a prebuild script